pkg/services/serviceaccounts/api/api.go

```diff
@@ -269,11 +269,12 @@ func (api *ServiceAccountsAPI) ConvertToServiceAccount(ctx *models.ReqContext) r
 	if err != nil {
 		return response.Error(http.StatusBadRequest, "Key ID is invalid", err)
 	}
-	if err := api.store.MigrateApiKey(ctx.Req.Context(), ctx.OrgId, keyId); err == nil {
-		return response.Success("Service accounts converted")
-	} else {
+
+	if err := api.store.MigrateApiKey(ctx.Req.Context(), ctx.OrgId, keyId); err != nil {
 		return response.Error(http.StatusInternalServerError, "Error converting API key", err)
 	}
+
+	return response.Success("Service accounts converted")
```

Comment:
naming convention is hard 🤣 

```suggestion
	return response.Success("Service accounts migrated")
```